src/Vinyl-s3.js

```diff
@@ -25,49 +27,69 @@ class VinylS3 {
     this.s3 = new AWS.S3({ region: options.region });
   }
 
-  src(globs, relativePath) {
+  /**
+   *
```

Comment:
Nit-picky, but some methods are missing the descriptions.

Reply:
https://dev.azure.com/techsmarts/Web%20Producer/_workitems/edit/718